spdx-utils/build.gradle.kts

```diff
@@ -81,27 +81,27 @@ val importLicenseTexts by tasks.registering {
 
 fun getLicenseHeader(fromYear: Int = 2017, toYear: Int = Year.now().value) =
     """
-        /*
-         * Copyright (C) $fromYear-$toYear HERE Europe B.V.
-         *
-         * Licensed under the Apache License, Version 2.0 (the "License");
-         * you may not use this file except in compliance with the License.
-         * You may obtain a copy of the License at
-         *
-         *     http://www.apache.org/licenses/LICENSE-2.0
-         *
-         * Unless required by applicable law or agreed to in writing, software
-         * distributed under the License is distributed on an "AS IS" BASIS,
-         * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-         * See the License for the specific language governing permissions and
-         * limitations under the License.
-         *
-         * SPDX-License-Identifier: Apache-2.0
-         * License-Filename: LICENSE
-         */
-
-
-     """.trimIndent()
+    |/*
+    | * Copyright (C) $fromYear-$toYear HERE Europe B.V.
+    | *
+    | * Licensed under the Apache License, Version 2.0 (the "License");
+    | * you may not use this file except in compliance with the License.
+    | * You may obtain a copy of the License at
+    | *
+    | *     http://www.apache.org/licenses/LICENSE-2.0
+    | *
+    | * Unless required by applicable law or agreed to in writing, software
+    | * distributed under the License is distributed on an "AS IS" BASIS,
+    | * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    | * See the License for the specific language governing permissions and
+    | * limitations under the License.
+    | *
+    | * SPDX-License-Identifier: Apache-2.0
+    | * License-Filename: LICENSE
+    | */
+
+
```

Comment:
I would prefer to add "|" to these empty lines as well, because it looks more consistent. Same several times below.

Reply:
Done.

Reply:
Thanks.